global.json

```diff
@@ -3,13 +3,18 @@
     "allowPrerelease": true
   },
   "tools": {
-    "dotnet": "7.0.304",
+    "dotnet": "8.0.100-preview.3.23178.7",
+    "runtimes": {
+        "dotnet": [
+            "8.0.0-preview.7.23363.2"
+        ]
+    },
```

Comment:
This is an Arcade custom thing that I wouldn't use if not strictly necessary. Why do you want to bring in the runtime separately from the SDK?

Reply:
It was introduced by https://github.com/dotnet/msbuild/pull/8672/commits/a54e08e73e95186b4a3135286df37cd622d4ec1e.
@rainersigwald can you help to answer?

Reply:
I think that at the time I was trying to use Arcade 8 but still target .NET 7. We shouldn't need this now--let me try removing it.